src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/model/CamundaProcessVariables.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.waworkflowapi.clients.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static uk.gov.hmcts.reform.waworkflowapi.clients.model.DmnValue.booleanValue;
+import static uk.gov.hmcts.reform.waworkflowapi.clients.model.DmnValue.dmnStringValue;
+
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@EqualsAndHashCode
+@ToString
+public class CamundaProcessVariables {
```

Comment:
Can you please add pojo tests for all the new pojos.

Reply:
added